tests/EdFi.Ods.Api.IntegrationTests/OrganizationDepartmentTests.cs

```diff
@@ -243,16 +243,14 @@ public void When_updating_organization_departments_parent_elsewhere_in_hierarchy
             
             EducationOrganizationHelper.ShouldContainTuples(Connection, expectedTuplesAfterMovedDown);
             
-            afterMovedDownCount.ShouldBe(afterCreationCount + 6,
-                () =>
-                {
-                    return $"The following additional tuples were found: {Environment.NewLine}"
-                    + string.Join(Environment.NewLine, 
-                        EducationOrganizationHelper.GetExistingTuples(Connection)
+            afterMovedDown.Count.ShouldBe(afterCreationCount + 6,
```

Comment:
Shouldly v4 deprecated `Func<string> customMessage overloads`. We have a roundtrip to the DB to build the error message, however, the existing variable `afterMovedDown` already has the values that we need, so I used it instead.